cocos2d/core/base-nodes/CCNodeCanvasRenderCmd.js

```diff
@@ -359,7 +363,20 @@ cc.Node.RenderCmd.prototype = {
             }
 
             if (node._additionalTransformDirty) {
-                this._transform = cc.affineTransformConcat(t, node._additionalTransform);
+                var additionalTransform = node._additionalTransform;
+                lScaleX = additionalTransform.a;
+                lScaleY = additionalTransform.d;
+                additionalTransform.a = 1;
+                additionalTransform.d = 1;
```

Comment:
The additionalTransform variable is reference to the node._additionalTransform so this will affect its value.